### ISSUE TITLE
New admin theme vs custom themes?

### ISSUE THREAD

**Issue by sreichel** [OpenMage/magento-lts#1164]
### Summary (*)

I've tested the admin theme (#1008) only with vanilla LTS and it worked like a charm.  After updating some shops to latest LTS i had to realize that **all** shops made changes to admin templates. Instead of changing default templates, they added a new theme that is based on default ... (seems correct to me)

```
    <stores>
        <admin>
            <design>
                <package>
                    <name>default</name>
                </package>
                <theme>
                    <default>default</default>
                    <template>some_name</template>
                </theme>
            </design>
        </admin>
    </stores>
```

So "openmage admin theme" is never loaded ... 

- it shows default login page (default templates loaded)
- only loads default css 

### Examples (*)

See above.

### Proposed solution

- add "openmage" to fallback logic? NO! (???)

or

- replace default admin theme with  #1008
- remove admin theme switcher

It is just CSS and should break nothing.

**Comment by aterjung** [comment]
ok, this behavior is intended to be shure custom themes are not affected. The shops i am supporting all use the default theme so i did not see this as a problem. Did you check if the themes would break if you change the base theme to openmage?

**Comment by sreichel** [comment]
> Did you check if the themes would break if you change the base theme to openmage?

Yes, i did. It shows openmage theme, but the storeswitcher does not work anymore.

**Comment by sreichel** [comment]
Do we still need the `default` theme, or can we (@aterjung) replace it? Actually it does not work with common (?) template rewrites.

**Comment by Flyingmana** [comment]
We can not replace it for backwards compatibility reasons, we could just copy it, which would increase the needed maintenance

**Comment by aterjung** [comment]
I am not shure if I understand the Problem! If there is a custom theme, isn’t it right to ignore the openmage theme?
Is the Problem that the theme switch dose not switch to openmage theme then? Is this really a problem?




Diplom Wirtschaftsinformatiker (FH)

**Comment by kiatng** [comment]
@aterjung An example to the problem: I have a custom template in `app\design\adminhtml\default\ihe\template\sales\order\view\history.phtml`, where I set my custom theme called `ihe`, where I added the admin user name to show which comment is made by who. Now, if I want to use the `openmage` theme, how do I fallback to it?

As suggested by @sreichel, one way is to replace the `default` theme with `openmage`. 

Another way is make a new package for `openmage`, I actually prefer this, for infinite theme fallback.
